### ISSUE TITLE
How do i actually run the examples?

### ISSUE THREAD

**Issue by yaadhavraajagility** [bazelbuild/rules_python#493]
bazel run //examples/pip_install:main

fails with 

ERROR: Skipping '//examples/pip_install:main': no such package 'examples/pip_install': Package is considered deleted due to --deleted_packages

**Comment by hrfuller** [comment]
We add them to -deleted-packages so the tests in them aren't run by `bazel test //...`. The examples are run as integration tests in CI, if you would like to run them outside of tests copy the contents of the example to a new directory outside of rules python. You will also have to update the rules_python version in the examples WORKSPACE file to point to a newer release.

**Comment by thundergolfer** [comment]
@hrfuller is correct that the examples are in a separate, embedded `WORKSPACE` and thus you must `cd` into the workspace root and run `bazel run //:main`. 

I copied out the `pip_install` and tried to run it and prior to `0.3.0` is had a problem where it was attempting to use `compile_pip_requirements` that wasn't available in `0.2.0`. You would get the following error: 

```
ERROR: error loading package '': cannot load '@rules_python//python/pip_install:requirements.bzl': no such file
```

now that `0.3.0` includes that file the error is fixed.

I think we should in future try to keep the examples in a working state and not add features into them that won't work.

----

@yaadhavraajagility do you have anything else you'd like addressed before closing this issue?
